index.js

```diff
@@ -13,6 +15,11 @@ var nums = {
 };
 
 module.exports = function (strNum) {
+  // remove comma if any
+  strNum = strNum.toString().search(',') ? strNum.toString().split(',').join('') : strNum;
```

Comment:
Could we avoid search and potential split/join by simpling using `strNum.replace(/,/g, '')`?

Reply:
updated :)